### ISSUE TITLE
vectorized lower_bound

### ISSUE THREAD

**Issue by HelliceSaouli** [boostorg/compute#860]
Hello
I have to sorted vectors(a, b) and i would like to do some binary search for each element in b i would like to have a lower_bound from a. this is called vectorized binary search which is possible in thrust, but i can't find it in boost

**Comment by chetanpandey1266** [comment]
Are you talking about this [link](https://thrust.github.io/doc/group__vectorized__binary__search_gac35a115642366fff80e85de54a9882c6.html#gac35a115642366fff80e85de54a9882c6)

**Comment by HelliceSaouli** [comment]
yes, that one. I end up implementing my own brute force custom kernel for it.  but it would be nice to have boost version
